src/core/public/application/ui/app_router.tsx

```diff
@@ -64,9 +64,9 @@ export const AppRouter: FunctionComponent<Props> = ({
               key={mounter.appRoute}
               path={mounter.appRoute}
               exact={mounter.exactRoute}
-              render={({ match: { url } }) => (
+              render={({ match: { path } }) => (
                 <AppContainer
-                  appPath={url}
```

Comment:
Note that I did not fix the legacy route below. Changing it caused a lot of test failures + no apps are currently using this and it is slated for removal in #74911

Reply:
Can't find proper documentation for it, what is the actual difference between `match.url` and `match.path` ?

Reply:
From the docs:
> path - (string) The path pattern used to match. Useful for building nested <Route>s
> url - (string) The matched portion of the URL. Useful for building nested <Link>s

From what I can gather, `path` will always be the path that was passed to the Route component that matched, while `url` will be the portion of the URL that matched. I think that can include the `/`, whereas our `path` never has a trailing slash unless the application explicitly includes on in the `appRoute` parameter.